compute/admin_guide/deployment_patterns/performance_planning.adoc

```diff
@@ -10,8 +10,18 @@ For example, heavily loaded hosts have fewer available resources than hosts with
 === Scale
 
 Prisma Cloud has been tested and optimized to support up to 10,000 Defenders per Console.
+
+ifdef::compute_edition[]
 Higher numbers of Defenders per Console can be supported, as long as the xref:../install/system_requirements.adoc#hardware[required resources] are allocated to Console.  
+endif::compute_edition[]
+
+
+ifdef::compute_edition[]
+=== Storage
+
+Using a network based storage is not recommended because it affects the database performance. if you choose to use a network based storage, such as NFS, make sure to review the https://www.mongodb.com/docs/v4.2/administration/production-notes/#remote-filesystems-nfs[Mongodb documentation] for NFS storage requirements.
```

Comment:
@MorShabiPAN, your feedback is in. Can you please help me state what we recommend using for storage, if not network based, is it local storage?

Reply:
Thanks Matangi